src/main/java/ezra/WrongFormatException.java

```diff
@@ -0,0 +1,16 @@
+package ezra;
+
+/**
+ * Exception thrown when there is an incorrect format in user input commands.
+ */
+public class WrongFormatException extends Exception {
```

Comment:
I like that you put this exception as checked, this way you will always be reminded to handle this exception.